### ISSUE TITLE
Ability to add channels without logging in

### ISSUE THREAD

**Issue by Rex000** [Seldszar/Gumbo#153]
IT would be great to have the ability to add channels and games without logging in tot twitch.

**Comment by Seldszar** [comment]
The Twitch API requires a token in order to work, and the extension was designed to work with an authenticated user in mind and use their token to interact with the API.

However, it could be possible to include a non-followed channel list in addition to the usual followed one (from Twitch), without having to break the extension's core design.

**Comment by Rex000** [comment]
Yes, that's what I had in mind actually. Logging in just to use the extension is fine. Only that I want to be able to add and remove channels and categories in an offline list.
